edit-post/components/meta-boxes/index.js

```diff
@@ -7,24 +7,13 @@ import { withSelect } from '@wordpress/data';
  * Internal dependencies
  */
 import MetaBoxesArea from './meta-boxes-area';
-import MetaBoxesPanel from './meta-boxes-panel';
 
-function MetaBoxes( { location, isActive, usePanel = false } ) {
+function MetaBoxes( { location, isActive } ) {
```

Comment:
I wondered about that too. After reading, I favor keeping `MetaBoxes` and `MetaBoxesArea` as-is. Since `MetaBoxesArea` saves a node ref and does some other DOM manipulation, it seems cleaner for it to be free of the question "Did I render something?"